### DISCUSSION TITLE
Add way to rewrite the docs to a different location / path

### DISCUSSION THREAD

**Discussion by stantios** [withastro/starlight#1008] [Feature Requests]
### What version of `starlight` are you using?

0.11.2

### What is your idea?

My idea is to add way to put the docs in a different location / path. This allows you to do alot of things like having each language in its own subdirectory rather than English being in the root directory and the other languages having a folder for each langauge. This can allow you to serve English Translations under '/' rather than '/en' which I think some users would prefer.

### Why is this feature necessary?

I think this feature is neccessary because it helps folders and files be more organised by serving them in a different folder rather than a dedicated directory. As I said before, it would help people that are doing translation work and help each language have its own folder.

### Do you have examples of this feature in other projects?

Yes! This feature is implemented in Mkdocs where the user is allowed to pick where the docs are loaded from in the website as shown [here](https://www.mkdocs.org/user-guide/configuration/#docs_dir). This link shows that you can choose a custom location on where to host and serve the docs rather than a set directory.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Will let people discuss here, but just as a workaround, I believe some people have used symlinks to achieve this. Basically, if you have your preferred folder structure, you then symlink those into `src/content/docs/` in the structure Starlight expects.
